frontend/src/components/profiles/AdminProfile.js

```diff
@@ -1,13 +1,88 @@
-import React from 'react';
+import React from 'react'
+import Item from './Item'
+import { Avatar,Box,Button,TextField } from '@material-ui/core'
+import '../../App.css'
```

Comment:
unnecessary import

App.css affects globally, we don't have to import it anywhere other than App.js